### ISSUE TITLE
您好，想问问单卡3090的超参数您试过没没啊

### ISSUE THREAD

**Issue by SC-shendazt** [hancyran/RepSurf#22]
您好，想问问单卡3090的config超参数您试过没没啊，由于我没得4张显卡，所以特向您请教

**Comment by hancyran** [comment]
你好，
我在3090上尝试过，可以跑的。你可以尝试改[这一行](https://github.com/hancyran/RepSurf/blob/main/segmentation/scripts/s3dis/train_pointnet2.sh#L11)去实现单卡的运行。感谢你的关注。

Hi,
I have tried the implementation on 3090, and it works as well. You could try changing [this line](https://github.com/hancyran/RepSurf/blob/main/segmentation/scripts/s3dis/train_pointnet2.sh#L11) in the script for segmentation, and so as for classification.

**Comment by SC-shendazt** [comment]
> 你好， 我在3090上尝试过，可以跑的。你可以尝试改[这一行](https://github.com/hancyran/RepSurf/blob/main/segmentation/scripts/s3dis/train_pointnet2.sh#L11)去实现单卡的运行。感谢你的关注。
> 
> Hi, I have tried the implementation on 3090, and it works as well. You could try changing [this line](https://github.com/hancyran/RepSurf/blob/main/segmentation/scripts/s3dis/train_pointnet2.sh#L11) in the script for segmentation, and so as for classification.

·······················································································································································································
Following your tips, I successfully run it, I am running the ISPRS dataset, but the single card 3090 does not run very well, could you tell me what are the key parameters of this code? The following are my configuration parameters, thanks



![360截图17001017526799](https://user-images.githubusercontent.com/74888508/217115430-aae53456-e032-4c59-b940-f774104e4aad.jpg)
![360截图16720331558462](https://user-images.githubusercontent.com/74888508/217115441-efd2e9a5-26d8-44cc-8dcc-bbb7822c9144.jpg)

**Comment by hancyran** [comment]
I am not familiar with ISPRS dataset, but I think you can follow some common protocals: to scale the batch size by #gpus, e.g., 8 for 4 gpus and 2 for 1 gpu, and to scale the learning rate by a scaled batch size, e.g., 0.006 for 2, and 0.012 for 4. You can also tune the hyperparameters of augmentation and voxel size, which mainly depends on data, but it takes time and I am not 100% sure that it may be useful. Besides, more complete input and more input points could be benenficial (the best option is to set voxel_max to None), but take care of the OOM issue.
